cockroachcloud/egress-perimeter-controls.md

```diff
@@ -0,0 +1,324 @@
+---
+title: Egress Perimeter Controls for CockroachDB dedicated (Preview)
+summary: Learn how to configure Egress Perimeter Controls for enhanced network security on a {{ site.data.products.dedicated }} cluster.
+toc: true
+toc_not_nested: true
+docs_area: security 
+---
+
+{% include feature-phases/preview-opt-in.md %}
+
+This page describes how Egress Perimeter Controls can enhance the security of {{ site.data.products.dedicated }} clusters, and gives an overview of how to manage a cluster's egress rules.
+
+## Why use Egress Perimeter Controls
+
+{{ site.data.products.dedicated }} clusters must access external resources for many purposes:
+
+- Managing backups as part of a disaster recovery plan
+- Using [Change data capture (CDC) changefeeds](../{{site.versions["stable"]}}/change-data-capture-overview.html)
+- Exporting data
+- Exporting logs
+
+By default, clusters can access external resources via the internet without restriction. This potentially leaves a cluster open to a *data exfiltration* scenario, wherein an attacker, often a [malicious insider](https://www.cisa.gov/defining-insider-threats), steals data by sending backups, changefeeds, data, or logs to a source that they control. 
+
+Operators of {{ site.data.products.dedicated }} clusters can mitigate against this risk by using Egress Perimeter Controls, which enable [admins](../{{site.versions["stable"]}}/security-reference/authorization.html#admin-role) to restrict egress to a list of specified external destinations. This adds a strong layer of protection against malicious data exfiltration. Along with other measures such as [Private Clusters](private-clusters.html), Egress Perimeter Controls are an important component in an overall strategy for maximizing network security.
+
+Further reading: [review how CockroachDB products differs in advanced security features](../{{site.versions["stable"]}}/security-reference/security-overview.html).
+
+
+{{site.data.alerts.callout_info}}
+Regardless of user-specific Egress Perimiter Control policy, egress is always permitted to services that are managed by Cockroach Labs and are essential to your cluster's functionality.
```

Comment:
@abhinavg6 updated your suggestion to "policies". Is that correct?

Reply:
Actually ''policy" is relevant Matt, because a policy is composed of multiple egress rules. The plural may confuse people that there are multiple policies across egress rules.